config/sidekiq.yml

```diff
@@ -1,3 +1,5 @@
+:dynamic: true
```

Comment:
I think this is not necessary anymore, since we now use Sidekiq to delay and not Sidekiq-scheduler

Reply:
I removed the line in the last commit